api/envoy/admin/v3/config_dump.proto

```diff
@@ -134,6 +138,9 @@ message ListenersConfigDump {
     DynamicListenerState draining_state = 4;
 
     // Set if the last update failed, cleared after the next successful update.
+    // The error_state field contains the rejected version of this particular
```

Comment:
Nit: `*error_state*`. Same in other comments below.

Reply:
Done.